CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## staged
 
+- Add native power flow solver by using `compute_pf(data_math)`
```

Comment:
mmm, should we rename this to compute_mc_pf, to be consistent with solve_mc_pf?

Reply:
agreed